graphql/util.lua

```diff
@@ -11,6 +11,16 @@ local function map(t, fn)
   return res
 end
 
+local function map_name(t, fn)
+  local res = {}
+  for _, v in ipairs(t or {}) do
```

Comment:
ipairs works only for arrays. And it confuses because "map" (several lines above) function uses pairs.

Reply:
This function creates map from array where map keys are from 'name' field of array element.

Reply:
Well, since it is a `for _, v` cycle creating a map, there shouldn't be any difference if we replace it with `pairs` for consistency

Reply:
Fixed